manager/orchestrator/task.go

```diff
@@ -58,8 +59,9 @@ func RestartCondition(task *api.Task) api.RestartPolicy_RestartCondition {
 	return restartCondition
 }
 
-// IsTaskDirty determines whether a task matches the given service's spec.
-func IsTaskDirty(s *api.Service, t *api.Task) bool {
+// IsTaskDirty determines whether a task matches the given service's spec and
```

Comment:
@anshulpundir this is not a blocker, but I think this comment could be more clear. More specifically, describing the condition under which it returns true/false.